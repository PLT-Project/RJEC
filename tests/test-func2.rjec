func isEven (x int, y bool) bool {
    if (x % 2 == 0 && y) {
        return true;
    } else {
        return false;
    }
    return true;
}

func foo (x int) {
    printi(x % 4);
}

func main() {
    printb(isEven(2, true));
    printb(isEven(3, true));
    printb(isEven(4, false));

    printi(foo(3));
}