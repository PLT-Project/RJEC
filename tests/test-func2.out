true
false
false
3
0